Update comment (#3981)
infiniflow/ragflow: docker/README.md

### Maximum file size

- `MAX_CONTENT_LENGTH`  

>   The maximum file size for each uploaded file, in bytes. You can uncomment this line if you wish to change the 128M file size limit. Please also update `client_max_body_size` in nginx/nginx.conf correspondingly.

Reviewer: ```suggestion
  The maximum file size for each uploaded file, in bytes. You can uncomment this line if you wish to change the 128M file size limit. After making the change, ensure you update `client_max_body_size` in nginx/nginx.conf correspondingly.
```
